src/model/src/DUST_EMIS.F

```diff
@@ -837,7 +838,7 @@ subroutine get_dust_emis( jdate, jtime, tstep, rjacm, cellhgt,
  
                 rlay1hgt = rjacm ( c,r ) / cellhgt
  
-                dust_em( c,r ) = dust_alpha * vflux * rlay1hgt *tfa(c,r) * tfb(c,r)
+                dust_em( c,r ) = dust_alpha * vflux * rlay1hgt! *tfa(c,r) * tfb(c,r)
```

Comment:
"*tfa(c,r) * tfb(c,r) " removed. Are you sure?

Reply:
because tfa and tfb not calculated in the code, so not able to call them here